src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java

```diff
@@ -35,6 +35,8 @@
 		SQL_BLACKLIST.add(Pattern.compile("^ANALYZE\\s+TABLE", Pattern.CASE_INSENSITIVE));
 		SQL_BLACKLIST.add(Pattern.compile("^SET\\s+PASSWORD", Pattern.CASE_INSENSITIVE));
 		SQL_BLACKLIST.add(Pattern.compile("^(CREATE|DROP|RENAME)\\s+USER", Pattern.CASE_INSENSITIVE));
+
+		SQL_BLACKLIST.add(Pattern.compile("^TRUNCATE\\s+TABLE", Pattern.CASE_INSENSITIVE));
```

Comment:
Need another rule to catch truncate of partitions similar to:

                SQL_BLACKLIST.add(Pattern.compile("^ALTER\\sTABLE.*TRUNCATE"));

For SQL such as ALTER TABLE foo TRUNCATE PARTITION p4;